test/test_jit.py

```diff
@@ -14148,6 +14148,34 @@ def test(tensor, generator):
 
             FileCheck().check_not("prim::PythonOp").run(cu.test.graph)
 
+    def test_parse_generator(self):
+        seed = 2024
```

Comment:
iiuc seed can be in [-2^63, 2^64) or perhaps [0, 2^64). Can you test some large seeds to see if the parser works?

e.g.:
* if you serialize a torchscript graph for a generator with a negative seed, is its value remapped to a positive value?
* can the parser work with a seed > 2^64 ?

Reply:
Technically seed can be `[0, 2^64)`. However, the scalar parser only supports signed integers. So, the parser is constrained to `[0, 2^63)`. However, that is good enough for my purposes. I've added tests for these cases